packages/core/src/StateT/index.ts

```diff
@@ -2,4 +2,67 @@
 
 import "../Operator/index.js"
 
-export * from "./Classic/index.js"
+import { pipe } from "../Function"
+import * as DSL from "../PreludeV2/DSL/index.js"
+import * as P from "../PreludeV2/index.js"
+
+interface StateT<A, B> {
```

Comment:
Had to do this otherwise type inference for `A` param was not working when using chain. Please let me know If there Is there a better way to fix this?

Reply:
very strange indeed, also to avoid errors down the line it is good to export all the used symbols